.github/workflows/build-containers.yml

```diff
@@ -0,0 +1,80 @@
+name: Build Containers
```

Comment:
Once the first workflow runs, this name stays indefinitely. So if we want to rename, better do it before this is merged.